src/decision/src/subroutines/Subroutine.cpp

```diff
@@ -16,7 +16,8 @@ void Subroutine::startup() {
 
     std::string topic   = private_nh.resolveName("output");
     uint32_t queue_size = 10;
-    publisher_ = private_nh.advertise<geometry_msgs::Twist>(topic, queue_size);
+    publisher_ =
+    private_nh.advertise<geometry_msgs::TwistStamped>(topic, queue_size);
```

Comment:
Your call if you want to do this here, but we can fix this wierd lack of indenting by removing this line from the `.clang-format` file: `ContinuationIndentWidth: '0'`